discretize/CylMesh.py

```diff
@@ -614,6 +588,12 @@ def vol(self):
 
     ####################################################
     # Active and Hanging Edges and Faces
+    #
+    #    - to find the active edges, faces, we use
+    #      krons of bools (sorry). It is more efficient
+    #      than working with 3D matrices. For an
+    #      example, see the comments in `_ishangingFx`
```

Comment:
Can we have the code example in here showing the equivalency? The note from the PR discussion between you and I should do well.

Reply:
yep, for sure